### PR TITLE
Bump Citus Enterprise version to 9.3.4

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,13 @@
+citus-enterprise (9.3.4.citus-1) stable; urgency=low
+
+  * Fixes a bug that could cause crashes with certain compile flags
+
+  * Fixes a bug with lists of configuration values in ALTER ROLE SET statements
+
+  * Fixes deparsing for queries with anonymous column references
+
+ -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Wed, 22 Jul 2020 9:0:20 +0000
```
Comment (onurctirtir) [inline debian/changelog]:
```suggestion
 -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Wed, 22 Jul 2020 9:00:20 +0000
```

**Review by hanefi — APPROVED** [review]
:shipit: all tests look good

**Comment by onurctirtir** [comment]
Testing debian/buster package locally, so not merging yet

**Comment by onurctirtir** [comment]
closing this pr :)

**Pull request closed without merging**